HISTORY.md

```diff
@@ -27,8 +27,6 @@
 
   * Add Mish activation function (#2158).
 
-  * Better error handling of eigendecompositions and Cholesky decompositions
```

Comment:
Could you undo this change, I don't think you were supposed to delete that. Add you function below this.